core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp

```diff
@@ -0,0 +1,433 @@
+            "base": false,
+            "configs": [
+              {
+                "name": "dfs_federation_namenode_nameservice",
+                "value": "nameservice1"
+              },
+              {
+                "name": "dfs_namenode_quorum_journal_name",
+                "value": "nameservice1"
+              },
+              {
+                "name": "dfs_http_port",
+                "value": "20101"
+              },
+              {
+                "name": "namenode_config_safety_valve",
+                "value": "\n        <property>\n          <name>dfs.namenode.redundancy.considerLoad</name>\n          <value>false</value>\n        </property>"
+              },
+              {
+                "name": "dfs_https_port",
+                "value": "20102"
+              }
+            ]
+          },
+          {
+            "refName": "hdfs-NAMENODE-2",
+            "roleType": "NAMENODE",
+            "base": false,
+            "configs": [
+              {
+                "name": "dfs_federation_namenode_nameservice",
+                "value": "nameservice1"
+              },
+              {
+                "name": "dfs_namenode_quorum_journal_name",
+                "value": "nameservice1"
+              },
+              {
+                "name": "autofailover_enabled",
+                "value": "true"
+              },
+              {
+                "name": "dfs_http_port",
+                "value": "20101"
+              },
+              {
+                "name": "dfs_https_port",
+                "value": "20102"
+              }
+            ]
+          },
+          {
+            "refName": "hdfs-DATANODE-BASE",
+            "roleType": "DATANODE",
+            "base": true,
+            "configs": [
+              {
+                "name": "dfs_datanode_failed_volumes_tolerated",
+                "value": "0"
+              },
+              {
+                "name": "dfs_data_dir_list",
+                "value": "/dfs/dn"
+              }
+            ]
+          },
+          {
+            "refName": "hdfs-BALANCER-BASE",
+            "roleType": "BALANCER",
+            "base": true
+          },
+          {
+            "refName": "hdfs-FAILOVERCONTROLLER-BASE",
+            "roleType": "FAILOVERCONTROLLER",
+            "base": true
+          },
+          {
+            "refName": "hdfs-JOURNALNODE-BASE",
+            "roleType": "JOURNALNODE",
+            "base": true,
+            "configs": [
+              {
+                "name": "dfs_journalnode_edits_dir",
+                "value": "/dfs/jn"
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "refName": "hive",
+        "serviceType": "HIVE",
+        "roleConfigGroups": [
+          {
+            "refName": "hive-HIVEMETASTORE-BASE",
+            "roleType": "HIVEMETASTORE",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "knox",
```

Comment:
There are 2 `"knox"` services in the template.  The other one also has `IDBROKER`, so I'd keep that one):

https://github.com/hortonworks/cloudbreak/blob/56876389d062599fa14e85df1afa1877ccc2f3c6/core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp#L403-L430

Reply:
You are right. It has to be removed. I found it in the test, I missed to remove it in the version that was published for review.